src/Redmine/Client.php

```diff
@@ -204,6 +182,94 @@ public function getUrl()
         return $this->url;
     }
 
+    /**
+     * HTTP GETs a json $path.
```

Comment:
too obvious to be mentioned, don't you think :)

Reply:
I removed all obvious docblocks and rephrase the descriptions. I would like to keep them for consistency.